protected/components/CustomGridView.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * CustomGridView Class
+ *
+ * Extends CGridView to add custom pagination styles by using bootstrap classes.
+ *
+ * Example usage:
+ * ```php
+ * $this->widget('CustomGridView', array(
+ *   'dataProvider' => $dataProvider,
+ *   // other options here...
+ * ));
+ * ```
+ *
+ * Note: This class assumes jQuery is loaded. It uses AJAX to refresh the pagination state.
+ */
+
+
+Yii::import('zii.widgets.grid.CGridView');
+
+class CustomGridView extends CGridView {
```

Comment:
extend widget so that it has bootstrap like styles for pagination in a reusable way (many admin tables will use this)